expr/functions/rangeOfSeries/function.go

```diff
@@ -34,10 +34,20 @@ func (f *rangeOfSeries) Do(ctx context.Context, e parser.Expr, from, until int64
 	if err != nil {
 		return nil, err
 	}
+	if len(series) == 0 {
+		return []*types.MetricData{}, nil
+	}
 
 	r := series[0].CopyName(e.Target() + "(" + e.RawArgs() + ")")
 	r.Values = make([]float64, len(series[0].Values))
 
+	commonTags := helper.GetCommonTags(series)
```

Comment:
May be double copy of tags (in CopyName and in GetCommonTags) not needed ?

Reply:
Nice catch, done [`9faff07` (#686)](https://github.com/go-graphite/carbonapi/pull/686/commits/9faff0746b138b1872c5c2bd60af72fda3f97613)